jobs/sql/extractions/all_encounters.sql

```diff
@@ -0,0 +1,108 @@
+SET @partition = '${partitionNum}';
+set sql_safe_updates = 0;
+set @next_appt_date_concept_id = CONCEPT_FROM_MAPPING('PIH', 5096);
+
+drop temporary table if exists temp_all_encounters;
+create temporary table temp_all_encounters
+(
+    encounter_id         int,
+    encounter_datetime   datetime,
+    patient_id           int,
+    visit_id             int,
+    creator              int(11),
+    user_entered         varchar(255),
+    encounter_location   varchar(255),
+    encounter_type_name  varchar(50),
+    entered_datetime     datetime,
+    emr_id               varchar(15),
+    next_appt_date       date,
+    voided               bit,
+    last_updated         datetime(3)
```

Comment:
Added last_updated date.  This essentially serves as our "watermark".  It will have the same value for all rows that are inserted in a particular incremental update.  So the max date in this column is our watermark date - the date as of which our data is up-to-date.  Note, this tracks the last updated date on dbevent, not the current date.  So if the last time data in the source DB was updated was a year ago, this will show a max of a year ago, not a max of the last time the pipeline ran.